aws/resource_aws_codebuild_webhook.go

```diff
@@ -0,0 +1,112 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/codebuild"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsCodeBuildWebhook() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsCodeBuildWebhookCreate,
+		Read:   resourceAwsCodeBuildWebhookRead,
+		Delete: resourceAwsCodeBuildWebhookDelete,
+		Update: resourceAwsCodeBuildWebhookUpdate,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
```

Comment:
This might seem pedantic for now, but on merge will rename this attribute to `project_name` to better align with the API and help operators understand the association with projects.